custom/icds_reports/utils/aggregation_helpers/distributed/deprecation.py

```diff
@@ -0,0 +1,161 @@
+from dateutil.relativedelta import relativedelta
```

Comment:
`deprecation.py` is a bit of a funny module name just because of the Python concept. maybe fine, just pointing it out.

Reply:
I think I may get rid of this entirely, and try to integrate this functionality into the existing agg helpers for the relevant steps, but if it sticks around, I can rename it. How does "location_reassignment" sound? That indicates it's functionality.

Reply:
"location_reassignment" sounds good to me :+1: